### DISCUSSION TITLE
Inquiry Regarding Modifying Barge FOWT Floating Platform Size in OpenFAST

### DISCUSSION THREAD

**Discussion by Cltsanity** [OpenFAST/openfast#1937] [Q&A]
Dear OpenFAST Community,

I hope this message finds you well. I am currently working on a project involving the modification of a ITI Energy Barge-based floating offshore wind turbine (FOWT) platform. Specifically, I am seeking advice and guidance on the feasibility and best practices for resizing the platform from its current dimensions of 40mx40m to 60mx60m within the OpenFAST framework.

I would greatly appreciate any insights, recommendations, or resources that the community can provide regarding:

Best regards,
Lingte
University of Glasgow

**Comment by jjonkman** [comment]
Dear @Cltsanity,

The existing OpenFAST model of the ITI Energy barge in the OpenFAST r-test (https://github.com/OpenFAST/r-test/tree/main/glue-codes/openfast/5MW_ITIBarge_DLL_WTurb_WavesIrr) considers the barge structurally as a rigid body and hydrodynamically as a potential-flow body with a crude approximation for viscous effects from a single strip-theory member.  To increase the size, naturally you'd want to change the rigid-body mass properties (mass, center of mass, inertias), and recalculate the potential-flow solution for the different size.  The viscous drag settings likely need updating, and of course, the mooring system dimensions would have change.  You'll also have to ensure that the model is set up consistently such that the weight of the turbine and barge balance with the vertical buoyancy and vertical mooring line pretension.

Best regards,
